### PR TITLE
Add MIT license

### PR DESCRIPTION
**Pull request by edunham**
https://github.com/cobalt-org/cobalt.rs/issues/40

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 E. Dunham
```
Comment (johannhof) [inline LICENSE]: Not sure you're the Copyright holder, maybe change it to "The Cobalt contributors"? :wink:

**Comment by johannhof** [comment]
Thanks for starting this, we really need a license. See above comment, apart from that good to merge for me.

**Comment by edunham** [comment]
Sorry about that, autogenerated the file on my fork.

**Comment by johannhof** [comment]
No worries, thanks again!

**Pull request merged**